app/models/response_map.rb

```diff
@@ -126,7 +127,7 @@ def self.participant_scores(participant, questions)
     # Retrieve assignment score
     compute_assignment_score(participant, questions, scores)
     # Compute the Total Score (with question weights factored in)
-    scores[:total_score] = assignment.compute_total_score(scores) 
+    scores[:total_score] = AssignmentHelper.compute_total_score(assignment, scores) 
```

Comment:
can you include assignment helper and not need to actually call it as AssignmentHelper?